.gitignore

```diff
@@ -1,33 +1,40 @@
-HELP.md
-target/
-!.mvn/wrapper/maven-wrapper.jar
-!**/src/main/**/target/
-!**/src/test/**/target/
+.gradle
```

Comment:
add ignore "target" folder

Reply:
Added .target to .gitignore file.

Reply:
this folder has to "disappear" from git at all:
![image](https://github.com/user-attachments/assets/b705e0c6-2178-4080-8af0-5e13be79e980)

Reply:
Made sure it is completely ignored using .gitignore.

Reply:
That was tricky. I guess I needed to revert and stop tracking the target folder.